phoenix-core/src/main/java/org/apache/phoenix/memory/GlobalMemoryManager.java

```diff
@@ -159,6 +161,10 @@ protected void finalize() throws Throwable {
         }
 
         private void freeMemory() {
+//        	System.out.println("Free memory! " + size);
```

Comment:
I think this a mistake, missed from your code clean-up.

Reply:
Done